CustomerManagementPanel/src/app/addresses/addresses.component.html

```diff
@@ -16,25 +16,32 @@
         </tr>
         </thead>
         <tbody>
-        <tr *ngFor="let address of addressesList; index as i">
+        <tr *ngFor="let address of orderedAddresses; index as i">
```

Comment:
After paging you have property address.orderedNumber, you dont need index anymore.